dojo/tools/pmd/parser.py

```diff
@@ -0,0 +1,60 @@
+import io
+import csv
+import hashlib
+from dojo.models import Finding
+
+
+class PmdParser(object):
+
+    def get_scan_types(self):
+        return ["PMD Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "CSV Report"
+
+    def get_findings(self, filename, test):
+        dupes = dict()
+
+        content = filename.read()
+        if type(content) is bytes:
+            content = content.decode('utf-8')
+        reader = csv.DictReader(io.StringIO(content), delimiter=',', quotechar='"')
+        csvarray = []
+
+        for row in reader:
```

Comment:
Could you get ride of the double loop?
I know some parser do it but it's not a good thing :/ 
I'm sure you can do something better :)